### PR TITLE
getExecutionNodes response and request

### PR DESCRIPTION
**Pull request by pariparajuli**
changed the proto

### CODE REVIEW

**Comment by tylerwilliams** [comment]
Here's a good article on how to write a good commit message: https://www.xspdf.com/resolution/2290016.html

For this CL, as an example, you could say:

Define GetExecutionNodes API request and response.

This change adds the Request and Response protos for the GetExecutionNodes API. It does not yet define the gRPC service or implementation, which will come in a followup CL.

File: proto/execution_stats.proto [hunk 1]
```diff
@@ -106,6 +106,29 @@ message Execution {
...
+  // Assignable memory bytes in remote execution node
+  int64 assignable_memory_bytes = 3;
+
+  // Assignable cpu  in remote execution node
+  int64 assignable_milli_cpu = 4;
+
+  // Remote execution node operating system
+  string os = 5;
+
+  string arch = 6;
```
Comment (tylerwilliams) [inline proto/execution_stats.proto:126]: nit: please add comments for the arch and pool fields.

Arch is the architecture of the remote execution node.
Pool is the remote execution pool that this node is assigned to.

**Review by tylerwilliams — CHANGES_REQUESTED** [review]
This looks good overall -- just a few minor nits and we'll be good to go!

File: proto/execution_stats.proto [hunk 2]
```diff
@@ -106,6 +106,29 @@ message Execution {
   string command_snippet = 7;
 }
 
+message ExecutionNode {
+
```
[the remark this answers is not in the document]
Comment (pariparajuli) [reply proto/execution_stats.proto:110]: Done

File: proto/execution_stats.proto [hunk 3]
```diff
@@ -106,6 +106,29 @@ message Execution {
...
+  // Assignable cpu  in remote execution node
+  int64 assignable_milli_cpu = 4;
+
+  // Remote execution node operating system
+  string os = 5;
+
+  string arch = 6;
+
+  string pool = 7;
```
[the remark this answers is not in the document]
Comment (pariparajuli) [reply proto/execution_stats.proto:128]: Done!

File: proto/execution_stats.proto [hunk 4]
```diff
@@ -106,6 +106,29 @@ message Execution {
...
+
```
[the remark this answers is not in the document]
Comment (pariparajuli) [reply proto/execution_stats.proto:129]: Done

File: proto/execution_stats.proto (hunk 1 above)
Comment (pariparajuli) [reply proto/execution_stats.proto:126]: Done.

**Review by tylerwilliams — APPROVED** [review]
LGTM

**Pull request merged**